### PR TITLE
Bug Fixes and Enhancements

### PR DESCRIPTION
**Pull request by RogueElement**
This should address issues:
#116, #117, #118, #119, #120, #122, #123, #124 and #126

### CODE REVIEW

**Comment by raedah** [comment]
ACK I have tested this and updated the related issues. Can be merged following code review.

File: views/root.tmpl [hunk 1]
```diff
@@ -356,7 +356,7 @@
                 </table>
             </div>
             <div class="col-md-6 text-left">
-                <div>Number of Tickets Mined in Last Block: <strong id="feeinfo_number">{{.StakeSummary.Feeinfo.Number}}</strong></div>
+                <div>Tickets in Lastest Block: <strong id="feeinfo_number">{{.StakeSummary.Feeinfo.Number}}</strong></div>
```
Comment (gozart1) [inline views/root.tmpl]: Should be Latest, not Lastest

File: views/tx.tmpl [hunk 2]
```diff
@@ -50,13 +70,24 @@
...
+                            N/A
+                        {{else}}
+                            <a href="/explorer/block/{{$.Block.BlockHash}}">{{.BlockHeight}}</a></td>
+                        {{end}}
                         </td>
-                        <td>{{if ne .BlockHeight 0}}{{.BlockHeight}}{{end}}</td>
                         <td>{{.Sequence}}</td>
                         <td>{{.Tree}}</td>
-                        <td>{{.AmountIn | getAmount}}</td>
+                        <td>{{if eq .AmountIn -1e-8}} N/A {{else}} {{.AmountIn}} {{end}}</td>
```
Comment (chappjc) [inline views/tx.tmpl]: Unless we understand clearly where -1e-8 is coming from (is it hard-coded somewhere?) I'd prefer if we change this to <0, but it's OK for now.

File: web.go [hunk 3]
```diff
@@ -177,6 +187,31 @@ func (td *WebUI) RootPage(w http.ResponseWriter, r *http.Request) {
...
+	str, err := TemplateExecToString(td.errorTempl, "error", struct {
+		ErrorCode   string
+		ErrorString string
+	}{
+		code,
+		msg,
+	})
+	if err != nil {
+		http.Error(w, "template execute failure", http.StatusInternalServerError)
+		return
```
Comment (chappjc) [inline web.go]: need to log `err`

**Comment by chappjc** [comment]
The "dcrdata.org" link in the footer goes to the wrong place.  I'm not sure where it should go tbh as we don't have an general organization page, and only a basic GH page for the dcrdata repository at http://www.dcrdata.org/

**Pull request merged**